doc/dialects/exasol.md

```diff
@@ -170,11 +170,28 @@ IDENTIFIED BY '<password>';
 
 The Exasol SQL dialect supports all capabilities that are supported by the virtual schema framework.
 
-## Known limitations
+## Known Limitations
 
-* Using literals and constant expressions with `TIMESTAMP WITH LOCAL TIME ZONE` data type in Virtual Schemas can produce an incorrect results.
-   * We recommend using `TIMESTAMP` instead.
-   * If you are willing to take the risk and want to use `TIMESTAMP WITH LOCAL TIME ZONE` anyway, please, create a Virtual Schema with the following additional property `IGNORE_ERRORS = 'TIMESTAMP_WITH_LOCAL_TIME_ZONE_USAGE'`.
-   * We also recommend to set Exasol system `time_zone` to UTC while working with `TIMESTAMP WITH LOCAL TIME ZONE`.
-* When using an EXA connection, the outermost order of the imported result rows is not guaranteed. This is not a bug, but a deliberate speed optimization in the ExaLoader (the part that runs the `IMPORT`) caused by parallel import.
-  If you need ordering, please wrap your query into an extra `SELECT * FROM (<virtual-schema-query>) ORDER BY <criteria> [, ...]` which will then be executed on the target Exasol database instead of the source.
+### Data type `TIMESTAMP WITH LOCAL TIME ZONE`
+
+Using literals and constant expressions with `TIMESTAMP WITH LOCAL TIME ZONE` data type in Virtual Schemas can produce an incorrect results.
+* We recommend using `TIMESTAMP` instead.
+* If you are willing to take the risk and want to use `TIMESTAMP WITH LOCAL TIME ZONE` anyway, please, create a Virtual Schema with the following additional property `IGNORE_ERRORS = 'TIMESTAMP_WITH_LOCAL_TIME_ZONE_USAGE'`.
+* We also recommend to set Exasol system `time_zone` to UTC while working with `TIMESTAMP WITH LOCAL TIME ZONE`.
+
+### Clause `ORDER BY`
+
+Clause `ORDER BY` can be used without limitations when using [local connection](#using-is_local).
+
+Connections [EXA](#using-import-from-exa) and [JDBC](#using-import-from-jdbc) are using `IMPORT` which only supports unordered data transfer. Therefore the outermost order of the imported result rows is not guaranteed.
+
+If you need ordering then please
+* apply the ordering on top-level in the Exasol target database outside the virtual schema,
+* use a sub-query to access the virtual schema and
+* annotate the sub-query with `ORDER BY FALSE` to prevent push down of the top-level `ORDER BY`, e.g.
+
+```sql
+SELECT * FROM (<virtual-schema-query> ORDER BY FALSE) ORDER BY <criteria> [, ...]
+```
+
+See also [General Known Limitations](https://docs.exasol.com/db/latest/database_concepts/virtual_schemas.htm#KnownLimitations) in official Exasol documentation on Virtual Schemas.
```

Comment:
I hope you are sure the information will be under that URL. :-)

Reply:
The URL is not confirmed, yet. But you can still follow the link and get to the page.
Is this acceptable or do you prefer a different solution?

Reply:
:+1: